src/components/ui/AppLoader/index.tsx

```diff
@@ -1,7 +1,7 @@
-import { Component } from 'react';
+import { Component, ReactElement } from 'react';
 import classnames from 'classnames';
-
-import injectStyle from 'react-jss';
+import withStyles, { WithStylesProps } from 'react-jss';
+import { Theme } from 'src/themes';
```

Comment:
Please always use relative file paths - that's the standard we have been using till date.

Reply:
will follow relative path format